libraries/lib-strings/Identifier.h

```diff
@@ -232,5 +232,7 @@ struct CommandIdTag;
 using CommandID = TaggedIdentifier< CommandIdTag, false >;
 using CommandIDs = std::vector<CommandID>;
 
+using ManualPageID = wxString;
```

Comment:
Good, I see moving to interface isolation (minimizing dependencies from frameworks types).
Also, I didn't see it in the previous commit with `ShowHelp()`, etc. I wait for something like this (as you do it here):

`HelpSystem.h`
```cpp
using WindowHandle = wxWindow; // minimizing dependencies from frameworks.
ShowHelp(wxWindow *parent, ...);
```
or move this typedefs to one file for get more isolation like:
`PlatformDefs/WxDefines.h` // declare here various: using `WindowHandle = wxWindow;`

you  know, but also, I see, that this is preliminaries and you will doing new PRs with things something like this.

Reply:
Something like a window handle will be in a later PR, yes.

Notice that this commit only make a type alias for wxString.    You could still pass wxString where ManualString is required, without error -- because they are the same type.

But the next commit makes ManualPageID a really different type so compilation would fail in such cases.

Before I make that change in the next commit, I make all the changes of argument declarations that will be necessary.  This is a first long and boring commit doing that, so that the really interesting thing in the next commit isn't lost among too many other changes.